### PR TITLE
Feature/recipe

### PR DESCRIPTION
**Pull request by no-or**

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "start": "nodemon server.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "body-parser": "^1.19.0",
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "mongoose": "^5.7.7"
+  },
```
Comment (DConnorFong) [inline server/package.json]: Can you please add the prettier extension, as well as TSLint. I have a top level lint config that you can reference.

File: recipe/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "recipe",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "npm run build:live",
+    "build": "tsc -p .",
```
Comment (DConnorFong) [inline recipe/package.json]: Change this to `tsc -p ./dist` I believe it should hide the compiled files which we shouldn't be committing.

File: recipe/.env [hunk 3]
```diff
@@ -0,0 +1,4 @@
+DB_CONNECTION=mongodb+srv://dood1:6iCdoaNy551UMgMx@main-cluster-vjjaz.mongodb.net/RecipeDB?retryWrites=true&w=majority
+PORT = 3000
```
Comment (DConnorFong) [inline recipe/.env]: Go to the drive under dev/ENV update dev.env and use that instead (note your changes in Changelog).

Also update this and the package.json to folder /config/dev.env. Its better code practice than committing your DB username and password to a public repository

File: recipe/package.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "recipe",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node --inspect=5858 -r ts-node/register ./src/index.ts",
```
Comment (DConnorFong) [inline recipe/package.json]: @no-or to use the config, install env-cmd like the other services and put `env-cmd -f ../config/dev.env` in here at the beginning.

**Comment by DConnorFong** [comment]
@no-or there are still JS commits in the PR. Can you remove them. Then we will be good to merge.

**Comment by no-or** [comment]
@DConnorFong I have noticed the JS files still being pushed. I took note of that and wanted to ask you in the next meeting how to fix that.

**Pull request merged**